geco/mips/packing.py

```diff
@@ -0,0 +1,79 @@
+import pyscipopt as scip
+from networkx.utils import py_random_state
+
+
+@py_random_state(-1)
+def tang(n, m, binary=False, seed=0):
+    """Generates a packing instance as described in A.2 in [1].
+
+    Parameters:
+    ----------
+        n: int
+            number of variables
+        m: int
+            number of constraints
+        seed: int, random state or None
+            seed for randomization
+
+    References:
+    ----------
+    .. [1] Tang, Y., Agrawal, S., & Faenza, Y. (2019). Reinforcement learning for integer
+    programming: Learning to cut. arXiv preprint arXiv:1906.04859.
+    """
+    return packing(n, m, *tang_params(n, m, seed), binary, name="Tang Packing")
+
+
+@py_random_state(-1)
+def tang_params(n, m, seed=0):
+    costs = [seed.randint(1, 10) for _ in range(n)]
```

Comment:
randint is [a,b) isn't it? So we need randint(1,10+1)

Reply:
it's [a,b] in the [docs](https://docs.python.org/3/library/random.html#random.randint)

Reply:
Oh, mh. It's [a,b) for numpy. Is this a problem?